### PR TITLE
pacific: doc/releases: update pacific release notes

### PR DESCRIPTION
**Pull request by zdover23**
This commit updates the Pacific release notes, which were not present until now.

Signed-off-by: Zac Dover <zac.dover@gmail.com>

## Contribution Guidelines
- To sign and title your commits, please refer to [Submitting Patches to Ceph](https://github.com/ceph/ceph/blob/main/SubmittingPatches.rst).

- If you are submitting a fix for a stable branch (e.g. "pacific"), please refer to [Submitting Patches to Ceph - Backports](https://github.com/ceph/ceph/blob/master/SubmittingPatches-backports.rst) for the proper workflow.

## Checklist
- Tracker (select at least one)
  - [ ] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [x] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [x] No impact that needs to be tracked
- Documentation (select at least one)
  - [x] Updates relevant documentation
  - [ ] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [x] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
</details>

### CODE REVIEW

**Review by colemitchell — APPROVED** [review]

File: doc/releases/pacific.rst [hunk 1]
```diff
@@ -2,6 +2,41 @@
...
+
+  With this hotfix, the vulnerability is fixed. Administrators who are
+  concerned they may have been impacted should audit the CephX keys in
+  their cluster for proper path restrictions.
+
+  Again, this vulnerability only impacts OpenStack Manila clusters which
+  provided native CephFS access to their users.
+
+* A regression made it possible to dereference a null pointer for
+  for s3website requests that don't refer to a bucket resulting in an RGW
```
Comment (colemitchell) [inline doc/releases/pacific.rst:30]: There's an extraneous _for_ on this line.

File: doc/releases/pacific.rst [hunk 2]
```diff
@@ -2,6 +2,41 @@
...
+Notable Changes
+---------------
+* Users who were running OpenStack Manila to export native CephFS, who
+  upgraded their Ceph cluster from Nautilus (or earlier) to a later
+  major version, were vulnerable to an attack by malicious users. The
+  vulnerability allowed users to obtain access to arbitrary portions of
+  the CephFS filesystem hierarchy, instead of being properly restricted
+  to their own subvolumes. The vulnerability is due to a bug in the
+  "volumes" plugin in Ceph Manager. This plugin is responsible for
+  managing Ceph File System subvolumes which are used by OpenStack
```
Comment (colemitchell) [inline doc/releases/pacific.rst:19]: @zdover23 Should there be a comma before _which_?

File: doc/security/CVE-2022-0670.rst [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+-------
+
+Users who were running OpenStack Manila to export native CephFS, who
+upgraded their Ceph cluster from Nautilus (or earlier) to a later
+major version, were vulnerable to an attack by malicious users. The
+vulnerability allowed users to obtain access to arbitrary portions of
+the CephFS filesystem hierarchy, instead of being properly restricted
+to their own subvolumes. The vulnerability is due to a bug in the
+"volumes" plugin in Ceph Manager. This plugin is responsible for
+managing Ceph File System subvolumes which are used by OpenStack
```
Comment (colemitchell) [inline doc/security/CVE-2022-0670.rst:16]: @zdover23 Should there be a comma before _which_?

File: doc/security/CVE-2022-0670.rst [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+Manila services as a way to provide shares to Manila users.
+
+Again, this vulnerability only impacts OpenStack Manila clusters which
```
Comment (colemitchell) [inline doc/security/CVE-2022-0670.rst:19]: @zdover23 I think this _which_ should be a _that_.

File: doc/security/CVE-2022-0670.rst [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+===========================================================
+
+Summary
+-------
+
+Users who were running OpenStack Manila to export native CephFS, who
+upgraded their Ceph cluster from Nautilus (or earlier) to a later
+major version, were vulnerable to an attack by malicious users. The
+vulnerability allowed users to obtain access to arbitrary portions of
+the CephFS filesystem hierarchy, instead of being properly restricted
```
Comment (zdover23) [inline doc/security/CVE-2022-0670.rst:13]: @colemitchell Do you agree that this comma should be removed?

Comment (colemitchell) [reply doc/security/CVE-2022-0670.rst:13]: @zdover23 Yes, I would agree.

File: doc/releases/pacific.rst [hunk 6]
```diff
@@ -2,6 +2,41 @@
...
+  Manila services as a way to provide shares to Manila users.
+
+  With this hotfix, the vulnerability is fixed. Administrators who are
+  concerned they may have been impacted should audit the CephX keys in
+  their cluster for proper path restrictions.
+
+  Again, this vulnerability only impacts OpenStack Manila clusters which
```
Comment (colemitchell) [inline doc/releases/pacific.rst:26]: @zdover23 I think this _which_ should be a _that_. Also, the _only_ should be relocated to after _impacts_.

File: doc/releases/pacific.rst [hunk 7]
```diff
@@ -2,6 +2,41 @@
...
+
+This is a hotfix release that resolves two security flaws.
+
+Notable Changes
+---------------
+* Users who were running OpenStack Manila to export native CephFS, who
+  upgraded their Ceph cluster from Nautilus (or earlier) to a later
+  major version, were vulnerable to an attack by malicious users. The
+  vulnerability allowed users to obtain access to arbitrary portions of
+  the CephFS filesystem hierarchy, instead of being properly restricted
```
Comment (colemitchell) [inline doc/releases/pacific.rst:16]: @zdover23 As per your previous comment, the comma in this line should be removed.

File: doc/releases/pacific.rst [hunk 8]
```diff
@@ -2,6 +2,41 @@
...
 =======
 
+v16.2.10 Pacific
+================
+
+This is a hotfix release that resolves two security flaws.
+
+Notable Changes
+---------------
+* Users who were running OpenStack Manila to export native CephFS, who
```
Comment (colemitchell) [inline doc/releases/pacific.rst:12]: @zdover23 I would remove the comma(s) and add an _and_ before _who upgraded_.

File: doc/security/CVE-2022-0670.rst [hunk 9]
```diff
@@ -0,0 +1,43 @@
+.. _CVE-2022-0670:
+
+CVE-2022-0670: Native-CephFS Manila Path-restriction bypass
+===========================================================
+
+Summary
+-------
+
+Users who were running OpenStack Manila to export native CephFS, who
```
Comment (colemitchell) [inline doc/security/CVE-2022-0670.rst:9]: @zdover23 I would remove the comma(s) and add an _and_ before _who upgraded_.

**Comment by zdover23** [comment]
@colemitchell: The suggestions that you have made for pacific.rst have been accepted and have been implemented in https://github.com/ceph/ceph/pull/48423.

**Comment by zdover23** [comment]
@colemitchell:
https://github.com/ceph/ceph/pull/48423
https://github.com/ceph/ceph/pull/48429

These two pull requests, together, make the grammatical changes that you have suggested here.

File: doc/releases/pacific.rst (hunk 2 above)
Comment (zdover23) [reply doc/releases/pacific.rst:19]: There should be, and now there is.

File: doc/security/CVE-2022-0670.rst (hunk 3 above)
Comment (zdover23) [reply doc/security/CVE-2022-0670.rst:16]: There should be, and now there is.

File: doc/security/CVE-2022-0670.rst (hunk 4 above)
Comment (zdover23) [reply doc/security/CVE-2022-0670.rst:19]: Both of these changes have been made.

File: doc/releases/pacific.rst (hunk 6 above)
Comment (zdover23) [reply doc/releases/pacific.rst:26]: Both of these changes have been made.

File: doc/releases/pacific.rst (hunk 7 above)
Comment (zdover23) [reply doc/releases/pacific.rst:16]: It has been removed.

File: doc/releases/pacific.rst (hunk 8 above)
Comment (zdover23) [reply doc/releases/pacific.rst:12]: Both of these changes have been made.

File: doc/security/CVE-2022-0670.rst (hunk 5 above)
Comment (zdover23) [reply doc/security/CVE-2022-0670.rst:13]: The comma has been removed.

File: doc/security/CVE-2022-0670.rst (hunk 9 above)
Comment (zdover23) [reply doc/security/CVE-2022-0670.rst:9]: These changes have been made.

**Pull request merged**